src/miniapp.ts

```diff
@@ -0,0 +1,5 @@
+export declare namespace MiniApp {
+  export function getUniqueId(): string;
```

Comment:
As discussed, we will need the functions in the JS SDK to be asynchronous due to iOS not supporting synchronously returning values. So the return value for this should be something like `Promise<string>`.